Woodoku/src/Entity/WoodokuBoard.py

```diff
@@ -55,11 +84,11 @@ def __validate(block: Tuple[int, int]) -> None:
         Raises:
             ShapeOutOfBoardError: when some block is not valid 
         """
-        pass
-    
-    
+        if not (0 <= block[0] <= N - 1 and 0 <= block[1] <= N - 1):
+            raise ShapeOutOfBoardError(block[0], block[1])
```

Comment:
```suggestion
		x, y = block
        if not (0 <= x <= N - 1 and 0 <= y <= N - 1):
            raise ShapeOutOfBoardError(x, y)
```
As a side note, try not to do index access in python for tuples. Unpack them. Here is another trick. `*block` is the same as `x, y`

to indeed you can also write line 88 as `raise ShapeOutOfBoardError(*block)`

Reply:
I see, but for readability, I might keep using x,y when raising the error